iree/vm/BUILD

```diff
@@ -186,6 +186,7 @@ cc_library(
         "bytecode_module.h",
     ],
     deps = [
+        ":c_funcs",
```

Comment:
I know I was probably the one who suggested this name, but was thinking that maybe we should rename this so it's clearer what they are (":ops"/":op_impls" or something) - there's a lot of "C functions" in iree/vm/, so not clear at first glance what a c_funcs.h is :)

Reply:
I agree :) Renamed to `ops.h`.